packages/autoprofile/compat/build-env/docker-compose.yml

```diff
@@ -9,9 +9,9 @@ services:
     volumes:
       - ../..:/opt/autoprofile
 
-  win-node-prebuild:
-    build:
-      context: .
-      dockerfile: Dockerfile-win
-    volumes:
-      - ../..:/opt/autoprofile
+#  win-node-prebuild:
+#    build:
+#      context: .
+#      dockerfile: Dockerfile-win
+#    volumes:
+#      - ../..:/opt/autoprofile
```

Comment:
@dmelikyan Is that intentionally commented out in this PR?

Reply:
compiling for windows on linux it isn't working yet, more time needed to make it work, if ever.